UPGRADE-7.1.md

```diff
@@ -50,17 +52,66 @@ DependencyInjection
  * [BC BREAK] When used in the `prependExtension()` method, the `ContainerConfigurator::import()` method now prepends the configuration instead of appending it
  * Deprecate `#[TaggedIterator]` and `#[TaggedLocator]` attributes, use `#[AutowireIterator]` and `#[AutowireLocator]` instead
 
+   *Before*
+   ```php
+   use Symfony\Component\DependencyInjection\Attribute\TaggedIterator;
+   use Symfony\Component\DependencyInjection\Attribute\TaggedLocator;
+
+   class HandlerCollection
+   {
+       public function __construct(
+           #[TaggedIterator('app.handler', indexAttribute: 'key')]
+           iterable $handlers,
+
+           #[TaggedLocator('app.handler')]
+           private ContainerInterface $locator,
+       ) {
+       }
+   }
+   ```
+
+   *After*
+   ```php
+   use Symfony\Component\DependencyInjection\Attribute\AutowireIterator;
+   use Symfony\Component\DependencyInjection\Attribute\AutowireLocator;
+
+   class HandlerCollection
+   {
+       public function __construct(
+           #[AutowireIterator('app.handler', indexAttribute: 'key')]
+           iterable $handlers,
+
+           #[AutowireLocator('app.handler')]
+           private ContainerInterface $locator,
+       ) {
+       }
+   }
+   ```
+
 DoctrineBridge
 --------------
 
- * Deprecated `DoctrineExtractor::getTypes()`, use `DoctrineExtractor::getType()` instead
+ * Deprecate `DoctrineExtractor::getTypes()`, use `DoctrineExtractor::getType()` instead
```

Comment:
This one should be removed actually https://github.com/symfony/symfony/commit/eb319a2cbfdcecb290f2023cee2ad524b305e20b

Reply:
fixed in ecd1d92161